CMakeLists.txt

```diff
@@ -19,25 +19,26 @@ if (NOT SMT_SWITCH_LIB_TYPE)
 endif()
 
 add_library(smt-switch "${SMT_SWITCH_LIB_TYPE}"
```

Comment:
Needed to add portfolio_solve here. Hope it's okay that I alphabetized the list while I was at it.

Reply:
Not at all, that's great!